sub-projects.md

```diff
@@ -0,0 +1,13 @@
+# ORAS Sub-Projects
+
+The ORAS Project represents the following sub-projects:
```

Comment:
Do we need to clarify the definition differences between **sub-project** and **repository**? This doc indicates that the **sub-project** is one or a group of repositories in the same category.